ostelco-core/Models/MyInfoDetails.swift

```diff
@@ -92,7 +92,7 @@ public struct MyInfoDetails: Codable {
         case _name = "name"
         case _dob = "dob"
         case _email = "email"
-        case address = "regadd"
+        case address = "mailadd"
```

Comment:
I think we should probably be passing a flag so that we can have this be `#if`-`#else`'d based on whether we're on dev or prod build. Otherwise this'll work on one but not the other.

Reply:
I think this renaming needs to happen in dev env also